### ISSUE TITLE
Fix calling node statistics with node specification

### ISSUE THREAD

**Issue by rapgru** [elefana/elefana#15]
Ticket #3 showed a problem with the elfana API where the endpoint `_nodes/<internal node id>/stats` is not available. The corresponding UAT is `testSingleNode`.

The endpoint `_nodes/<internal node id>/stats/os` returns a result, but doesn't have the expected form. The corresponding UAT is `testSingleNodeOsOnly`.

**Comment by rapgru** [comment]
The problem is that elefana doesn't distinguish between the [Node Stats API](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/cluster-nodes-stats.html) and the [Node Info API](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/cluster-nodes-info.html). Rather it returns the expected data of a node stats request when issuing a node info request.

**Comment by rapgru** [comment]
I changed the `ApiRouter`to return the node stats when a `stats` request is issued. This fixes the problem stated in this ticket.
However, I noticed that the node selection is processed and stored in `NodesInfoRequest`, but ignored when executing the request. This means that when sending a request for `_nodes/<internal node id>/stats` the url component `<internal node id>` has no effect. When given a non-existing node id, it still returns stats of all elefana instances.
**To fix this I need more information about the distributed version of elefana.**

**Comment by rapgru** [comment]
I changed the `ApiRouter` to return the node stats when a stats request is issued. This fixes the problem stated in this ticket.
However, I noticed that the node selection is processed and stored in `NodesInfoRequest`, but ignored when executing the request. This means that when sending a request for `_nodes/<internal node id>/stats` the url component `<internal node id>` has no effect. When given a non-existing node id, it still returns stats of all elefana instances.
**To fix this I need more information about the distributed version of elefana.**
